audit-ci.json

```diff
@@ -2,5 +2,5 @@
     "high": true,
     "pass-enoaudit": true,
     "retry-count": 20,
-    "allowlist": ["1067259", "node-forge"]
```

Comment:
What was this?

Reply:
It was a saml-js based CVE notification.    npm updated how it handles CVE noifications and the new version of audit-ci *only* takes URI based formats (iirc it's because that report number can change if a new alert is issued on the same issue).... however the issue no longer fails audit (manually/locally).